solana/ts/scripts/config/devnet/quoter.json

```diff
@@ -28,27 +28,27 @@
   "peerQuotes": {
     "Sepolia": {
       "maxGasDropoffEth": "0",
-      "basePriceUsd": "500000",
-      "nativePriceUsd": "3500000000",
+      "basePriceUsd": "1",
```

Comment:
shouldn't this be 0.5?

Reply:
I actually commited the wrong file, I wanted to commit mainnet